core/templates/domain/objects/UnitsObjectFactory.ts

```diff
@@ -190,7 +204,8 @@ export class UnitsObjectFactory {
     // Special symbols need to be replaced as math.js doesn't support custom
     // units starting with special symbols. Also, it doesn't allow units
     // followed by a number as in the case of currency units.
-    var keys = Object.keys(ObjectsDomainConstants.CURRENCY_UNITS);
+    var keys = <(keyof typeof ObjectsDomainConstants.CURRENCY_UNITS)[]>
```

Comment:
This type is used in many cases, probably extract it into a variable.

Reply:
done